GAE/src/com/gallatinsystems/gis/map/MapUtils.java

```diff
@@ -174,6 +175,11 @@ public static void recomputeCluster(Cache cache, SurveyedLocale locale, int delt
             }
         }
 
+        // delete locale if the Delta was a subtraction
+        SurveyedLocaleDao slDao = new SurveyedLocaleDao();
+        if (delta < 0) {
```

Comment:
The deletion only happens when the deleted `SurveyInstance` is the only one connected to the `SurveyedLocale`